app-pda/ideviceinstaller/ideviceinstaller-1.1.1.ebuild

```
@@ -0,0 +1,23 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A tool to interact with the installation_proxy of an Apple's iDevice"
+HOMEPAGE="http://www.libimobiledevice.org/"
+SRC_URI="https://github.com/libimobiledevice/ideviceinstaller/releases/download/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
```

```suggestion
LICENSE="GPL-2+"
```
see: https://github.com/libimobiledevice/ideviceinstaller/blob/master/src/ideviceinstaller.c#L11C1-L12C39